financial-connections/src/main/java/com/stripe/android/financialconnections/features/attachpayment/AttachPaymentViewModel.kt

```diff
@@ -60,15 +64,8 @@ internal class AttachPaymentViewModel @AssistedInject constructor(
                     params = PaymentAccountParams.LinkedAccount(requireNotNull(id))
                 )
             }
-            if (manifest.isNetworkingUserFlow == true && manifest.accountholderIsLinkConsumer == true) {
-                result.networkingSuccessful?.let {
-                    successContentRepository.set(
-                        message = PluralId(
-                            value = R.plurals.stripe_success_pane_desc_link_success,
-                            count = accounts.size
-                        )
-                    )
-                }
+            if (result.networkingSuccessful == true) {
```

Comment:
This was broken before, since `result.networkingSuccessful?.let { … }` would run even if `networkingSuccessful` was `false`.

Reply:
🤦